evm/ts/__tests__/calculateLocalTokenAddress.test.ts

```diff
@@ -0,0 +1,48 @@
+import { MultiTokenNtt } from "@wormhole-foundation/sdk-definitions-ntt";
+import { EvmMultiTokenNtt } from "../src/multiTokenNtt.js";
+import { toUniversal } from "@wormhole-foundation/sdk-definitions";
+
+describe("calculateLocalTokenAddress", () => {
```

Comment:
When a token does not exist yet on the destination chain (it gets created on the first transfer), it will be created at this address. The calculation is a bit involved, so added a test for it using real values from testnet.